Eval addition : Describe the meaning of Japanese onomatopoeia

a-c-jltd opened the pull request:
# Thank you for contributing an eval! ♥️

🚨 Please make sure your PR follows these guidelines, __failure to follow the guidelines below will result in the PR being closed automatically__. Note that even if the criteria are met, that does not guarantee the PR will be merged nor GPT-4 access granted. 🚨

__PLEASE READ THIS__:

In order for a PR to be merged, it must fail on GPT-4. We are aware that right now, users do not have access, so you will not be able to tell if the eval fails or not. Please run your eval with GPT-3.5-Turbo, but keep in mind as we run the eval, if GPT-4 gets higher than 90% on the eval, we will likely reject since GPT-4 is already capable of completing the task.

We plan to roll out a way for users submitting evals to see the eval performance on GPT-4 soon. Stay tuned! Until then, you will not be able to see the eval performance on GPT-4. **Starting April 10, the minimum eval count is 15 samples, we hope this makes it easier to create and contribute evals.**

Also, pelase note that we're using **Git LFS** for storing the JSON files, so please make sure that you move the JSON file to Git LFS before submitting a PR. Details on how to use Git LFS are available [here](https://git-lfs.com).

## Eval details 📑
### Eval name
Describe the meaning of Japanese onomatopoeia

### Eval description

The purpose of this Evals is to make you explain the meaning of Japanese onomatopoeia concisely.

### What makes this a useful eval?

Better Japanese expressions can be established.
We believe that onomatopoeia in Japanese is one of the most characteristic and important ways of expression in the Japanese language.
Currently, when you input Japanese onomatopoeia into GPT, it suggests meanings that do not exist or ways of expression that are not used in daily life.
(For example: the meaning of "zuki zuki" is described as a pulsating pain, but it is output as "the pain is felt tinglingly and gradually," etc.)

## Criteria for a good eval ✅

Below are some of the criteria we look for in a good eval. In general, we are seeking cases where the model does not do a good job despite being capable of generating a good response (note that there are some things large language models cannot do, so those would not make good evals).

Your eval should be:

- [x] Thematically consistent: The eval should be thematically consistent. We'd like to see a number of prompts all demonstrating some particular failure mode. For example, we can  create an eval on cases where the model fails to reason about the physical world.
- [x] Contains failures where a human can do the task, but either GPT-4 or GPT-3.5-Turbo could not.
- [x] Includes good signal around what is the right behavior. This means either a correct answer for `Basic` evals or the `Fact` Model-graded eval, or an exhaustive rubric for evaluating answers for the `Criteria` Model-graded eval.
- [x] **Include at least 15 high quality examples.**

If there is anything else that makes your eval worth including, please document it below.

### Unique eval value

> Insert what makes your eval high quality that was not mentioned above. (Not required)

## Eval structure 🏗️

Your eval should
- [x] Check that your data is in `evals/registry/data/{name}`
- [x] Check that your yaml is registered at `evals/registry/evals/{name}.yaml`
- [x] Ensure you have the right to use the data you submit via this eval

(For now, we will only be approving evals that use one of the existing eval classes. You may still write custom eval classes for your own cases, and we may consider merging them in the future.)

## Final checklist 👀

### Submission agreement

By contributing to Evals, you are agreeing to make your evaluation logic and data under the same MIT license as this repository. You must have adequate rights to upload any data used in an Eval. OpenAI reserves the right to use this data in future service improvements to our product. Contributions to OpenAI Evals will be subject to our usual Usage Policies (https://platform.openai.com/docs/usage-policies).

- [x] I agree that my submission will be made available under an MIT license and complies with OpenAI's usage policies.

### Email address validation

If your submission is accepted, we will be granting GPT-4 access to a limited number of contributors. Access will be given to the email address associated with the merged pull request.

- [x] I acknowledge that GPT-4 access will only be granted, if applicable, to the email address used for my merged pull request.

### Limited availability acknowledgement

We know that you might be excited to contribute to OpenAI's mission, help improve our models, and gain access to GPT-4. However, due to the requirements mentioned above and high volume of submissions, we will not be able to accept all submissions and thus not grant everyone who opens a PR GPT-4 access. We know this is disappointing, but we hope to set the right expectation before you open this PR.

- [x] I understand that opening a PR, even if it meets the requirements above, does not guarantee the PR will be merged nor GPT-4 access granted.

### Submit eval

- [x] I have filled out all required fields of this form
- [x] I have used **Git LFS** for the Eval JSON data
- [x] (Ignore if not submitting code) I have run `pip install pre-commit; pre-commit install` and have verified that `black`, `isort`, and `autoflake` are running when I commit and push

Failure to fill out all required fields will result in the PR being closed.

### Eval JSON data 

Since we are using Git LFS, we are asking eval submitters to add in as many Eval Samples (at least 5) from their contribution here:

<details>
  <summary>View evals in JSON</summary>

  ### Eval
  ```jsonl
  {"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "いがいが"}], "ideal": "とげのある刺激を表す。例文は「乾燥で喉がいがいがする。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "いじいじ"}], "ideal": "態度がはっきりしない様子。例文は「あいつはいじいじしてばっかりで、自分の意見を言わない。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "あたふた"}], "ideal": "慌てる様子。例文は「寝坊してあたふたする。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "あつあつ"}], "ideal": "熱い料理。例文は「あつあつのおでんを口に放り込まれる。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "あっさり"}], "ideal": "しつこくない様子。例文は「あっさりと高校生活が終わってしまった。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "キリキリ"}], "ideal": "鋭い痛みを表す。例文は「ストレスで胃がキリキリと痛む。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ギリギリ"}], "ideal": "限界に近い様子。例文は「ギリギリを攻めるな。余裕持て。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "キンキン"}], "ideal": "冷えている様子。例文は「冷凍庫にいれてキンキンにしたビール」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "キンコンカンコン"}], "ideal": "学校のチャイムの音。例文は「早く座らないとキンコンカンコン言っちゃうよ。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ぐいぐい"}], "ideal": "引っ張る様子。例文は「寂しそうに服の裾をぐいぐいと引っ張られる。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "グウグウ"}], "ideal": "空腹でお腹が鳴る音、もしくは居眠りするいびきの音。例文は「お昼休みはおなかがグウグウとなった。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "グキッ"}], "ideal": "関節の鳴る音。例文は「グキッと足首をひねった。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "くすくす"}], "ideal": "小さく笑う声。例文は「彼女はぼくをみてくすくすと笑った。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "クタクタ"}], "ideal": "とても疲れて力が抜けている様子。例文は「仕事のあとはいつもクタクタだ。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "くちゃくちゃ"}], "ideal": "咀嚼音。例文は「くちゃくちゃと音を立ててごはんを食べるな。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ぐちゃぐちゃ"}], "ideal": "整理できていない様子。例文は「色々ありすぎて、頭の中がぐちゃぐちゃだ。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "グツグツ"}], "ideal": "煮込む音、またはその様子。例文は「カレーをグツグツ煮込んだ。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ぐっすり"}], "ideal": "深い睡眠。例文は「今日はたくさん遊んだので、ぐっすり眠れそうだ。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ぐったり"}], "ideal": "力が抜ける様子。また、疲れて脱力している様子。例文は「暑さと疲れでぐったりとしている。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "くどくど"}], "ideal": "同じことをしつこく繰り返して言う様子。例文は「くどくどと文句を言って、みっともない。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "くねくね"}], "ideal": "曲がって続く様子。もしくは、体を柔らかく曲げる様子。例文は「田んぼの中にくねくねと踊る何かがいる。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "グビグビ"}], "ideal": "喉を鳴らしながら飲み物をのむ音。例文は「お風呂上りにビールをグビグビ飲む。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "クラクラ"}], "ideal": "めまいがして倒れそうになる様子。例文は「いきなり立ち上がって頭がくらくらした。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "グラグラ"}], "ideal": "安定しないで揺れ動く様子。例文は「歯がぐらぐらして今にも抜けそうだ。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "くるくる"}], "ideal": "小さなものが回る様子。例文は「レンギョウはくるくる回って落ちる。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ぐるぐる"}], "ideal": "大きなものが何度も回る様子。例文は「同じ道をぐるぐる回ってしまっている。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "クンクン"}], "ideal": "においを嗅ぐ様子、または音。例文は「救助犬はにおいをクンクンと嗅いで、人を救助します。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "グングン"}], "ideal": "勢いよく進行する様子。例文は「英語がぐんぐん上達した。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "げっそり"}], "ideal": "やせ衰える様子。例文は「疲れてげっそりとしている。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ゲラゲラ"}], "ideal": "大声で思いっきり笑う様子。例文は「テレビを見てゲラゲラと笑う。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ケロケロ"}], "ideal": "カエルの鳴き声。例文は「夜の田んぼはケロケロと鳴く声がうるさい。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ゴクゴク"}], "ideal": "喉を鳴らして勢いよく飲み物を飲む音。例文は「水をゴクゴクと飲んだ。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "コケコッコー"}], "ideal": "にわとりの泣き声。例文は「朝日が昇ると鶏はコケコッコーと鳴く。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "こそこそ"}], "ideal": "人に隠れて何かをする様子。例文は「こそこそと冷蔵庫を漁る。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ゴシゴシ"}], "ideal": "何かを強くこする際に出る音。例文は「バスタブをゴシゴシと洗う。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ごちゃごちゃ"}], "ideal": "物が乱雑に入り混じった様子。例文は「机の上がごちゃごちゃしている。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "コツコツ"}], "ideal": "地道に続けること、または指などで硬いものを叩く音。例文は「コツコツ努力したおかげで、優勝することができた。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ごつごつ"}], "ideal": "角ばっている様子。例文は「川上の方は石がごつごつしています。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ごっそり"}], "ideal": "たくさんの物がなくなる様子。例文は「髪の毛がごっそり抜けた。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "こってり"}], "ideal": "味が濃い様子。例文は「こってりの味噌ラーメンを食べる。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ごつん"}], "ideal": "重いものがぶつかる音。例文は「ごつんと鈍い音が響いた。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "コトコト"}], "ideal": "物を煮込む様子、音。例文は「カレーをコトコト煮込む。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "こなごな"}], "ideal": "細かく砕ける様子。例文は「ガラスがこなごなになった。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ゴホゴホ"}], "ideal": "咳の音。例文は「母が朝からゴホゴホしている。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "コロコロ"}], "ideal": "小さいものが転がる音。例文は「えんぴつを机の上でコロコロした。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ゴロゴロ"}], "ideal": "大きいものが転がる音。例文は「大きな岩がゴロゴロ転がる。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "コンコン"}], "ideal": "ノックの音。例文は「入るときはドアをコンコンして。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ザーザー"}], "ideal": "激しい雨音。例文は「雨がザーザーと降り、川が氾濫した。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "さっぱり"}], "ideal": "爽やかな様子や何も残っていない様子を表す。例文は「さっぱり忘れてしまった」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "サラサラ"}], "ideal": "べたつかない様子。また、葉っぱなどがこすれあう音。例文は「さらさらと笹が揺れている。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ザラザラ"}], "ideal": "手触りが荒い様子。例文は「猫の舌はザラザラしている。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "シトシト"}], "ideal": "静かに降る雨の音。例文は「シトシトと雨が降っている。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "じめじめ"}], "ideal": "湿気が多い様子。例文は「梅雨に入り、じめじめとした空気だ。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "じゃぶじゃぶ"}], "ideal": "勢いよく水をかき回す音。例文は「じゃぶじゃぶと水をかきわけた。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "シャカシャカ"}], "ideal": "素早く振る様子。また、その時の音。例文は「マラカスをシャカシャカ振る。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "シャキシャキ"}], "ideal": "歯ごたえのある食べ物を食べる音。例文は「シャキシャキのレタス」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "シュー"}], "ideal": "蛇の出す音。例文は「森の中でシューという音を聞いた。ハブかな。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ジュージュー"}], "ideal": "何かを焼く音。例文は「外で肉をジュージュー焼いた。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "しゅわしゅわ"}], "ideal": "細かい気泡が発生するときの音。例文は「炭酸でのどがしゅわしゅわした。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ジョリジョリ"}], "ideal": "髭や髪を剃る音。または、短い髭や髪に触る音。例文は「兄のあごがジョリジョリしている。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ジリジリ"}], "ideal": "太陽が強く照り付ける様子。また、暑さで焼かれているような感覚。例文は「肌の表面がジリジリする。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "じろじろ"}], "ideal": "遠慮なく見つめる様子。例文は「じろじろ見られている。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "しわしわ"}], "ideal": "しわが多い様子やふやけている様子。例文は「お風呂からあがると手がしわしわになる。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "じんじん"}], "ideal": "しびれるような痛みを表す。例文は「しもやけで手がじんじん痛む。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "スカスカ"}], "ideal": "隙間がたくさんある様子、空間が多い様子。例文は「このキャベツは中身がスカスカだ」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ズキズキ"}], "ideal": "脈打つような痛みを表す。例文は「頭がズキズキと痛む。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "すくすく"}], "ideal": "成長する様子。例文は「庭に植えた木がすくすくと育っている。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "スタスタ"}], "ideal": "しっかりした様子で歩く様子。例文は「振り向きもしないでスタスタ行ってしまった。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "すっきり"}], "ideal": "気持ちよく爽やかである様子。例文は「いらないものを捨ててすっきりした。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ずぶずぶ"}], "ideal": "水分が多い場所に深く沈み込む様子、音。例文は「ぬかるみにずぶずぶはまった。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "スベスベ"}], "ideal": "なめらかである様子。例文は「肌がスベスベになった。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "すやすや"}], "ideal": "静かに眠る様子。例文は「電車の中ですやすやと眠ってしまった。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "スラスラ"}], "ideal": "滞りなく進む様子。例文は「日本語をスラスラ読んだ。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "すりすり"}], "ideal": "何かをこすりつける音。例文は「寒くて手をすりすりした。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ズルズル"}], "ideal": "麺などを啜る音。例文は「パスタをズルズル啜ってはいけない。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ぞくぞく"}], "ideal": "寒気がする様子、興奮や恐怖で身震いする様子。例文は「素晴らしいパフォーマンスでぞくぞくした。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "そっくり"}], "ideal": "似ている様子。例文は「お父さんにそっくりだ。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "そっと"}], "ideal": "静かに何かをする様子。例文は「寝ている子供にそっと毛布をかけた。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "そよそよ"}], "ideal": "静かに吹く風を表す擬音語。例文は「そよそよと風が心地いい。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "そわそわ"}], "ideal": "落ち着かない様子。例文は「荷物が届かないかそわそわする。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "たっぷり"}], "ideal": "たくさんある様子。例文は「トーストにバターをたっぷり塗った。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "だぼだぼ"}], "ideal": "大きすぎて体形に合っていない様子。例文は「だぼだぼの服を買ってしまった。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "だらだら"}], "ideal": "気が緩んでいる様子。例文は「休日なので、家でだらだらする。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ちくちく"}], "ideal": "刺すような痛みを表す。例文は「指にとげが刺さってちくちくする。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "チャラチャラ"}], "ideal": "浮ついた様子。例文は「チャラチャラした男が向こうから歩いてきた。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "チューチュー"}], "ideal": "ねずみの泣き声。例文は「トイレからチューチューと声がする。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "チュンチュン"}], "ideal": "すずめの泣き声。例文は「窓からチュンチュンと泣き声が聞こえる。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "チョキチョキ"}], "ideal": "ハサミで切る音。例文は「紙をハサミでチョキチョキした。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ちょろちょろ"}], "ideal": "小さいものが動き回る様子。例文は「家の中をちょろちょろ動く。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ちらっと"}], "ideal": "瞬間的に見る様子。例文は「様子をちらっとみた。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "つーん"}], "ideal": "鼻の奥を突き刺すような刺激を表す。例文は「わさびを食べてつーんとした。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ツヤツヤ"}], "ideal": "光沢があり綺麗な様子。例文は「髪がツヤツヤだ。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "つるつる"}], "ideal": "なめらかで手触りが良い様子。また、よく滑る状態も表す。例文は「ガラスはつるつるしている」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "てきぱき"}], "ideal": "手際がよい様子。例文は「家の掃除をてきぱき行う。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "てくてく"}], "ideal": "ゆっくり長い距離を歩いている様子。例文は「駅からてくてく歩く。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "でこぼこ"}], "ideal": "表面が平らでなく起伏がある状態。例文は「地面がでこぼこだ。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "でれでれ"}], "ideal": "だらしのない様子。例文は「かわいい犬をみてでれでれする。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ドカーン"}], "ideal": "爆発音を表す擬音語。例文は「ドカーンという大きな音で目が覚めた。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ドキドキ"}], "ideal": "心臓の音、運動した後や恐れ、期待などの要因により心臓が激しく打つ様子。例文は「ホラー映画を見てドキドキした。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "とげとげ"}], "ideal": "とがったものという次の単語。例文は「ウニはとげとげしている。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "どばどば"}], "ideal": "勢いよく続けて出すという次の単語。例文は「砂糖をどばどばかけた。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "トロトロ"}], "ideal": "粘り気のある状態という次の単語。例文は「トロトロのオムライスを作った。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ドロドロ"}], "ideal": "粘り気のある流動物という次の単語。例文は「溶岩がドロドロと流れた。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "トントン"}], "ideal": "軽く叩くという次の単語。例文は「肩をトントンと叩いた。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "ドンドン"}], "ideal": "強く叩くという次の単語。例文は「ドアをドンドンと叩いた。」"}
{"input": [{"role": "system", "content": "次の単語の意味と例文を教えてください。正確かつ簡潔に答えてください。"}, {"role": "user", "content": "どんより"}], "ideal": "重苦しいという次の単語。例文は「どんよりとした空気だ。」" }
  ```
</details>

Ein-Tim commented:
Please fill in the `INSERT_EVAL_HERE` field of the PR template.

a-c-jltd commented:
> Please fill in the `INSERT_EVAL_HERE` field of the PR template.

Thanks for the heads up.

usama-openai reviewed (changes requested):
Thanks for the contribution.

The path of the samples file is provided incorrectly in the `.yaml` file. The samples file is located at `Japanese_onomatopoeia/samples.jsonl`, but the path provided in the `.yaml` file is `japanese_onomatopoeia/samples.jsonl`. Kindly correct this mistake.

We would love to review the PR again after the suggested changes.

a-c-jltd commented:
@usama-openai 
Thank you for your kind words. I have made the necessary corrections. Is it okay now?

usama-openai reviewed (changes requested):
Thanks for the contribution. I would like to request some changes.

1. Merge the `master` branch into your branch to resolve workflow related issues.
2. Due to the nature of this task, methods like `Includes`, `Match`, or `FuzzyMatch` will not work. For example, consider the following example:
    `ぐちゃぐちゃ`
    The ideal answer for this is `整理できていない`, and the model-provided answer is `「ぐちゃぐちゃ」は、ものが混ざり合ったり、汚れたりして、整然としていない様子を表す日本語の擬態語です`. Logically, the answer provided by the model is correct, but the evaluation method will mark this completion as wrong.

    I would recommend using the `ModelGraded` evaluation method, in which you ask the model to grade if the ideal answer and the completion are approximately correct. For this purpose, you can either implement custom model-graded spec or use the existing ones. You can read documentation or view other PRs to get a good idea of how to use the `ModelGraded` evaluation.

We would love to review the PR again after the suggested changes.

a-c-jltd commented:
@usama-openai 
I apologize for the repetition. Thank you for your advice. 
I have implemented it as you advised. What do you think?

usama-openai commented:
Thanks for implementing the requested changes.

Model grading is normally used to grade the model's completion against the ideal answer or against a certain criteria. In this case, the model is supposed to grade the completion against the ideal answer, but the ideal answer is not provided to the model for grading. The model-graded spec file is closely related to `closedqa`, but in my opinion, the `fact` kind of approach would be more suitable for this case.

a-c-jltd commented:
> 

I apologize for this late reply.
It took me a long time to output my thoughts. I referred to `fact` as advised. Then I added example sentences to `samples` ideal answers and also changed `modelgraded` significantly accordingly.

usama-openai reviewed (changes requested):
I'm getting the following error while evaluating this PR:
```
b'File "/content/evals/evals/eval.py", line 132, in eval_sample'
b'return idx, self.eval_sample(sample, rng)'
b'File "/content/evals/evals/elsuite/modelgraded/classify.py", line 82, in eval_sample'
b'choice, info = classify('
b'File "/content/evals/evals/elsuite/modelgraded/classify_utils.py", line 79, in classify'
b'choice = get_choice(evaluation, mg.eval_type or eval_type, match_fn, choice_strings)'
b'File "/content/evals/evals/elsuite/modelgraded/classify_utils.py", line 116, in get_choice'
b'if eval_type.startswith("cot_classify"):'
b"AttributeError: 'NoneType' object has no attribute 'startswith'"
```

This is happening because you have not specified the `eval_type` property in the `yaml` file.

Also, the spec file you are using doesn't need `args`, so you can remove the `modelgraded_spec_args` property from the `yaml` file.

a-c-jltd commented:
> I'm getting the following error while evaluating this PR:
> 
> ```
> b'File "/content/evals/evals/eval.py", line 132, in eval_sample'
> b'return idx, self.eval_sample(sample, rng)'
> b'File "/content/evals/evals/elsuite/modelgraded/classify.py", line 82, in eval_sample'
> b'choice, info = classify('
> b'File "/content/evals/evals/elsuite/modelgraded/classify_utils.py", line 79, in classify'
> b'choice = get_choice(evaluation, mg.eval_type or eval_type, match_fn, choice_strings)'
> b'File "/content/evals/evals/elsuite/modelgraded/classify_utils.py", line 116, in get_choice'
> b'if eval_type.startswith("cot_classify"):'
> b"AttributeError: 'NoneType' object has no attribute 'startswith'"
> ```
> 
> This is happening because you have not specified the `eval_type` property in the `yaml` file.
> 
> Also, the spec file you are using doesn't need `args`, so you can remove the `modelgraded_spec_args` property from the `yaml` file.

I am terribly sorry for the inconvenience. We have made the change and would appreciate your confirmation.

usama-openai reviewed (approved):
This PR looks in good shape now. I'm approving this PR.

a-c-jltd commented:
Thanks!

usama-openai commented:
You should see GPT-4 API access enabled in your account in the next few days.
